packages/reports/addon/templates/components/navi-column-config.hbs

```diff
@@ -8,6 +10,8 @@
           <NaviColumnConfig::Item
             @column={{column}}
             @isLastAdded={{eq column this.lastAddedItem}}
+            @isOpen={{eq column.fragment this.currentlyOpenFragment}}
```

Comment:
What is the thinking behind checking/storing `column.fragment` vs just the `column` obj?

Reply:
The fragments determine if we re-render the columns, for example changing the time grain would create a new column.
Since we're comparing only the fragment, no real reason to store the entire column.

Reply:
In general, I think it is nicer to use higher levels of abstraction if there is no benefit to using lower level concepts (like performance)? If we can avoid leaking the`fragment` concept, I think we should.